api/src/paths/project/{projectId}/attachments/upload.ts

```diff
@@ -142,72 +144,3 @@ export function uploadMedia(): RequestHandler {
     }
   };
 }
-
```

Comment:
is this test not needed anymore?

Reply:
Sorry dont exactly know what you are pointing at here